releasenotes/notes/default_otlp_logs_enabled_false-6d98a23f7a76f153.yaml

```
@@ -0,0 +1,11 @@
+# Each section from every release note are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+fixes:
+  - |
+    OTLP logs ingestion is now disabled by default, and otlp_config: logs: enabled: true must be specified to enable
```

```suggestion
    OTLP logs ingestion is now disabled by default. To enable it, set otlp_config.logs.enabled to true.
```
I think dot notation is a bit easier to read ^